polaris.shopify.com/src/components/IconGallery/IconGallery.tsx

```diff
@@ -1,21 +1,24 @@
 import Image from "../Image";
 import { useState } from "react";
-import TextField from "../TextField";
-import icons from "../../data/icons.json";
 import Fuse from "fuse.js";
+import { Tab } from "@headlessui/react";
+
+import {Icons} from '../../types';
+import TextField from "../TextField";
 import styles from "./IconGallery.module.scss";
 import Longform from "../Longform";
-import { Tab } from "@headlessui/react";
 import { className } from "../../utils/various";
+
+
+
+const icons: Icons = require('@shopify/polaris-icons/metadata');
 const importedSvgs = require.context(
   "../../../../polaris-icons/icons",
   true,
   /\.svg$/
 );
```

Comment:
I left this but the `@shopify/polaris-icons` package has a `dist/svg` directory where we should be able to access these files. I tried a couple of things and failed.

Reply:
> I tried a couple of things and failed.

You've added an `exports` key to the package.json. That limits what is visible to places that try to import things. I think you might need to add `dist/svg/*.svg`: `dist/svg/*.svg` or something to that to keep being able to access those files